src/fight.cpp

```diff
@@ -2156,21 +2170,29 @@ void hit(CharData *ch, CharData *victim, int type) {
          * Okay, we know the guy has been hit.   Now calculate damage,
          * starting with the damage bonuses: damroll and strength apply.
          */
-        dam = str_app[GET_STR(ch)].todam;
-        dam += GET_DAMROLL(ch);
+        dam = GET_DAMROLL(ch);
+        if (type != SKILL_KICK) {
+            dam += str_app[GET_STR(ch)].todam;
 
-        if (diceroll == 20)
-            dam += dam;
+            if (diceroll == 20)
+                dam += dam;
```

Comment:
While is this fine, `dam *= 2` is more expressive IMHO.  I also think it's faster, but the compiler will probably reduce it all down to a left shift anyway.